src/main/java/org/stellar/sdk/Util.java

```diff
@@ -103,62 +98,19 @@ public static String getSdkVersion() {
 
   public static ClaimableBalanceID claimableBalanceIdToXDR(String balanceId) {
     byte[] balanceIdBytes = BaseEncoding.base16().lowerCase().decode(balanceId.toLowerCase());
-    XdrDataInputStream balanceIdXdrDataInputStream =
-        new XdrDataInputStream(new ByteArrayInputStream(balanceIdBytes));
-
     try {
-      return ClaimableBalanceID.decode(balanceIdXdrDataInputStream);
+      return ClaimableBalanceID.fromXdrByteArray(balanceIdBytes);
```

Comment:
this seems too specific to claimablebalance to be in a generic util, and since you've added the xdr helper methods, how about refactoring this and deleting these claimable balance methods from Util and clients can just inline the one remaining line in the method for same result?

Reply:
Fixed in fb444200aaa92ce564c1a0a978c077becc29a423